master/buildbot/newsfragments/githubpullrequestpoller-secrets.bugfix

```diff
@@ -0,0 +1 @@
+``GitHubPullrequestPoller`` now supports secrets in its ``token`` argument
```

Comment:
Nitpick: append ```(:issue:`4921`) ``` to mention the bug number.

Reply:
Done!